### PR TITLE
feat: add support for setting default boot entry

### PR DESCRIPTION
**Pull request by OriginCode**
- Introduce `systemd-boot-conf` lib, use the lib to support systemd-boot `loader.conf`.
- Add support for setting a kernel as default boot entry.

@MingcongBai Could you please help review the translation files? お願い🙇

### CODE REVIEW

**Comment by OriginCode** [comment]
Also @NeoChen1024 could you please help review the zh_TW translation? 🙇

File: i18n/zh-CN/systemd_boot_friend_rs.ftl [hunk 1]
```diff
@@ -21,4 +21,6 @@ no_overwrite = 文件未作修改。
 overwrite = 正在覆盖 {$entry} ...
 create_entry = 正在 {$path} 建立 {$kernel} 启动项...
 remove_kernel = 正在移除 {$kernel} 内核文件...
-remove_entry = 正在移除 {$kernel} 启动项...
\ No newline at end of file
+remove_entry = 正在移除 {$kernel} 启动项...
```
Comment (MingcongBai) [inline i18n/zh-CN/systemd_boot_friend_rs.ftl]: 正在移除启动项 {$kernel}

File: i18n/en-US/systemd_boot_friend_rs.ftl [hunk 2]
```diff
@@ -21,4 +21,6 @@ no_overwrite = Doing nothing on this file.
 overwrite = Overwriting {$entry} ...
 create_entry = Creating boot entry for {$kernel} at {$path} ...
 remove_kernel = Removing {$kernel} kernel ...
-remove_entry = Removing {$kernel} boot entry ...
\ No newline at end of file
+remove_entry = Removing {$kernel} boot entry ...
+set_default = Setting {$kernel} as default boot entry ...
+ask_set_default = Do you want to set {$kernel} as default boot entry?
```
Comment (MingcongBai) [inline i18n/en-US/systemd_boot_friend_rs.ftl]: Would you like *

File: i18n/en-US/systemd_boot_friend_rs.ftl [hunk 3]
```diff
@@ -21,4 +21,6 @@ no_overwrite = Doing nothing on this file.
 overwrite = Overwriting {$entry} ...
 create_entry = Creating boot entry for {$kernel} at {$path} ...
 remove_kernel = Removing {$kernel} kernel ...
-remove_entry = Removing {$kernel} boot entry ...
\ No newline at end of file
+remove_entry = Removing {$kernel} boot entry ...
```
Comment (MingcongBai) [inline i18n/en-US/systemd_boot_friend_rs.ftl]: boot entry {$kernel}

File: i18n/zh-TW/systemd_boot_friend_rs.ftl [hunk 4]
```diff
@@ -1,5 +1,5 @@
 conf_default = {$conf_path} 不存在！正在產生模板...
-conf_old = 偵測到舊的配置檔，正在更新 ...
+conf_old = 偵測到舊的配置檔，正在更新...
```
Comment (NeoChen1024) [inline i18n/zh-TW/systemd_boot_friend_rs.ftl]: 配置檔 -> 設定檔

**Pull request merged**